### PR TITLE
store/tikv: update scanner

### PR DESCRIPTION
**Pull request by disksing**
@ngaut @morefreeze

### CODE REVIEW

**Comment by morefreeze** [comment]
Seems `scan_test.go` about L85 can be removed?

**Comment by morefreeze** [comment]
rest LGTM

File: store/tikv/scan.go [hunk 1]
```diff
@@ -54,6 +54,11 @@ func newScanner(region *requestRegion, startKey []byte, ver uint64, snapshot tik
 		region:       region,
 		skipFirst:    false,
 	}
+	err := scanner.Next()
```
Comment (shenli) [inline store/tikv/scan.go]: Why call next here?

**Comment by shenli** [comment]
LGTM

**Pull request merged**